src/Stratis.SmartContracts.Networks/SmartContractsPoARegTest.cs

```diff
@@ -38,7 +38,9 @@ public SmartContractsPoARegTest()
                 maxStandardVersion: 2,
                 maxStandardTxWeight: 100_000,
                 maxBlockSigopsCost: 20_000,
-                maxStandardTxSigopsCost: 20_000 / 5
+                maxStandardTxSigopsCost: 20_000 / 5,
+                federationPublicKeys: this.ConsensusOptions.FederationPublicKeys,
```

Comment:
You'd expect the keys to be inside this class rather than inherited from the base network class.
Which begs the question: do we need an instantiable PoaNetwork class at all?
am I right in thinking that the only difference in the number of properties (not their values) is the consensusOptions?

Reply:
>You'd expect the keys to be inside this class rather than inherited from the base network class.

it's a consequence of moving fed keys to network options. 
basically we need to copy all values from inherited network except for fed keys. 

it's the only way since those are protected- so we can only set whole new instance of consensus options. 

That's only of the reasons why I think it's better to have those in network itself.

Reply:
In Network rather than in ConsensusOptions?

Reply:
ugh, but ok